### PR TITLE
✨ Adjust sidebar and create footer

### PR DESCRIPTION
**Pull request by valdersonjr**
# ✨ Adjust sidebar and create footer

## Motivation
- Adjust sidebar
- Adjust footer

## Changes
- Images and descriptions added to the contributors section on sidebar.
- Added styling to the components of contributors (section of the sidebar).
- Creation of footer.
- Added styling to the footer.
- Images added to the contributors section on sidebar.
- Added styling to the components of the footer.

Screenshot from 2022-03-19 19-01-05
Screenshot from 2022-03-19 19-01-15
Screenshot from 2022-03-19 19-01-23

### CODE REVIEW

File: src/components/Modal/Contributors/Contributors.tsx [hunk 1]
```diff
@@ -1,11 +1,204 @@
...
+							</Styles.personData>
+						</Styles.personDatas>
+					</div>
+				);
+			})}
+
+			<br></br>
+			<h2>Developers:</h2>
+			<br></br>
+			{otherContributors.map((otherContributor, index) => {
```
Comment (LeoSilvaGomes) [inline src/components/Modal/Contributors/Contributors.tsx:166]: It's better you segregate this `otherContributors` in a array with `developers` and `researchers`. Why? Imagine new developers join in our team, we will need to update the index value all the time. So it's better have another map running into a array then this littles bugs for the future

File: src/components/Modal/Contributors/Contributors.tsx [hunk 2]
```diff
@@ -1,11 +1,204 @@
...
+									{otherContributor.institution}
+								</Styles.personData>
+							</Styles.personDatas>
+						</div>
+						{index == 1 && (
+							<>
+								<h2>Researchers:</h2>
+								<br></br>
+							</>
+						)}
```
Comment (LeoSilvaGomes) [inline src/components/Modal/Contributors/Contributors.tsx:196]: This is not maintainable

File: src/components/Modal/Contributors/styles.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+	padding: 70px;
+`;
+
+export const contributorImage = styled.img`
+	width: 200px;
+	height: 200px;
+	border-radius: 10px;
+`;
+
+export const avatarIconDiv = styled.div`
```
Comment (LeoSilvaGomes) [inline src/components/Modal/Contributors/styles.ts:13]: Tips - make some spaces here to improve the code readability

**Review by LeoSilvaGomes — COMMENTED** [review]
Add some images in the Pr description

File: src/components/Footer/Footer.tsx [hunk 4]
```diff
@@ -1,7 +1,19 @@
...
+	return (
+		<Styles.FooterContainer>
+			{institutionLogoImages.map((institution, index) => {
+				return (
+					<Styles.InstitutionsLogo
+						src={institution.logoImage}
+						alt={`Logo image of ${institution.name}`}
+					/>
+				);
+			})}
```
[the remark this answers is not in the document]
Comment (valdersonjr) [reply src/components/Footer/Footer.tsx:14]: Ooh, interesting! I wasn't aware of that.

File: src/components/Modal/Contributors/Contributors.tsx (hunk 1 above)
Comment (valdersonjr) [reply src/components/Modal/Contributors/Contributors.tsx]: Got it! I'll take a look into it.

**Pull request merged**